lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -218,6 +214,12 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         return this._tags;
     }
 
+    get tagsToDisplay(): Chip[] {
```

Comment:
Can we don’t do mapping in getter? Getter is called in html and we should avoid calling functions in html (so getter should not call map function). Instead this mapping should be done in place where tags are assigned.

Reply:
we could, but that would need changes in component to the scratch, the original property is passed to the others and that would need separate task